config/jobs/kubernetes-sigs/kernel-module-management/kernel-module-management-presubmits.yaml

```diff
@@ -12,6 +12,37 @@ presubmits:
       - image: golang:1.18
         command:
         - ci/prow/build
+  - name: pull-kernel-module-management-check-api-changes
+    always_run: true
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-node-kernel-module-management
+      testgrid-tab-name: check-api-changes
+      description: "make sure deployment manifests and bundle are up-to-date."
+    spec:
+      containers:
+      - image: docker.io/library/golang:1.19-alpine3.16
+        command: [sh]
+        args:
+        - -c
+        - |
+          apk add bash curl gcc git make
+
+          # Install kubectl
+          curl -Lo /usr/local/bin/kubectl "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+          chmod +x /usr/local/bin/kubectl
+
+          # install operator-sdk
+          curl -Lo /usr/local/bin/operator-sdk 'https://github.com/operator-framework/operator-sdk/releases/download/v1.25.2/operator-sdk_linux_amd64'
+          chmod +x /usr/local/bin/operator-sdk
+
+          # Include the destination directory of `go install` in $PATH
+          export PATH=$(go env GOPATH)/bin:${PATH}
+
+          ./ci/prow/check-api-changes
```

Comment:
Seems like we need a running cluster for `make manifests` - I am not sure we have a valid KUBECONFIG if we don't use the `kubekins-e2e` image.

Reply:
Why do we need a running cluster?

Reply:
I believe this can be resolved.